generator/generator.yml

```diff
@@ -108,6 +108,70 @@ modules:
       infeedVACapacityUsed:
         ignore: true  # Composite metric: infeedApparentPower / infeedVACapacity * 100
 
+# ServerTech Sentry 4 MIB
+#
+# Used by ServerTech PDUs
+#
+# https://cdn10.servertech.com/assets/documents/documents/816/original/Sentry4OIDTree.txt
+# https://cdn10.servertech.com/assets/documents/documents/815/original/Sentry4.mib
+  servertech_sentry4:
+    walk:
+      - sysUpTime
+      - 1.3.6.1.4.1.1718.4.1.3.3   # st4InputCordMonitorTable
+      - 1.3.6.1.4.1.1718.4.1.4.3   # st4LineMonitorTable
+      - 1.3.6.1.4.1.1718.4.1.5.3   # st4PhaseMonitorTable
+      - 1.3.6.1.4.1.1718.4.1.8.3   # st4OutletMonitorTable
+      - 1.3.6.1.4.1.1718.4.1.9.3   # st4TempSensorMonitorTable
+      - 1.3.6.1.4.1.1718.4.1.14.3  # st4FanSensorMonitorTable
+
+    lookups:
+      - source_indexes:
+         - st4UnitIndex
```

Comment:
You you keep these on one line like we have elsewhere in this file?

Reply:
will clean this these up

Reply:
refactored to single line to fit convention